README.md

```diff
@@ -1 +1,236 @@
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Html structure with Button, Inputs, etc]**
+- **[Styling css with colors, borders, paddings, margins & sizes, etc]**
+_ **[javascript functions, modules, ES6 syntax, variables, etc]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> I will use API to store user data in the Leaderboard Project.
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: to Clone the repo.
+<!--
+Example command:
+_git clone https://github.com/Atril33/Leaderboard.git
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+_ Opend CMD, Terminal or GitBash in your pc and type this
+ _git clone command _https://github.com/Atril33/Leaderboard.gitt_
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+https://github.com/Atril33/Todo_List_ES6.git
+```
+--->
+
+### Install
+
+Install this project with:
+_ You need to run this command _npm i_
+Then run this comman _npm run dev_
+<!--
+Example command:
+
+```sh
+  cd my-project
+  _git clone https://github.com/Atril33/Leaderboard.git_
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+_ You can use it by download in your computer.
+ This is this link for this project _https://github.com/Atril33/Leaderboard.git_
+<!--
+Example command:
+
+```sh
+  rails server
+### Deployment
+
+You can deploy this project using:
+Not Yet
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Syed</a>
+
+> This Project is created by Syed Atril
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- JAVASCRIPT
+# <a name="authors">Syed Atril</a>
+👤 **Syed Atril**
+
+- GitHub: [@githubhandle](https://github.com/Atril33)
+- Twitter: [@twitterhandle](https://twitter.com/AtrilSyed)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/syed-atril-831696248/)
+- Facebook [Facebook](https://web.facebook.com/profile.php?id=100073724910623)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```

Comment:
- Please, kindly remove this placeholder text. It was added to help you make updates to the template and is not expected to be in the readme after you've made the update.

Reply:
### PACKAGE.JSON FILE 


- [ ] Please, kindly add the Missing start to the script to make sure your project runs locally from the users' machine.

```js
"scripts": {
    "build": "webpack",
    "dev": "webpack serve",
    "start": "webpack serve --open"
  }
```